corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -868,7 +908,9 @@ def __init__(self, domain, registry_slug, source_type, source_id):
         registry_helper = DataRegistryHelper(self.domain, registry_slug=self.registry_slug)
         owning_domain = registry_helper.registry.domain
         prop_map = get_data_dict_props_by_case_type(owning_domain)
-        self.case_properties = sorted(set(prop_map[self.source_id]) | {'closed', 'closed_on'})
+        self.case_properties = sorted(
+            set(prop_map[self.source_id]) | {'closed', 'closed_on'} - {COMMCARE_PROJECT}
```

Comment:
`-` has higher precedence than `|`:
```python
>>> {'foo', 'bar', 'qux'} | {'foo', 'baz'} - {'qux'}
{'bar', 'baz', 'foo', 'qux'}
```
You probably want:
```python
>>> ({'foo', 'bar', 'qux'} | {'foo', 'baz'}) - {'qux'}
{'bar', 'baz', 'foo'}
```
i.e.:
```suggestion
            (set(prop_map[self.source_id]) | {'closed', 'closed_on'}) - {COMMCARE_PROJECT}
```

Reply:
Thanks, this isn't actually needed at all
```suggestion
            set(prop_map[self.source_id]) | {'closed', 'closed_on'}
```